middlewares/members.js

```diff
@@ -48,15 +48,17 @@ exports.updateUser = async (req, res) => {
     });
 };
 
-// TODO: reimplement by not deleting, but anonymizing a user.
```

Comment:
Is this still something you would want to add in the future?

Reply:
yep, we need the ability to soft-delete a user in the future anyway. This would probably be another endpoint, as it also should be accessible by user himself